test_conformance/api/negative_enqueue_map_image.cpp

```diff
@@ -29,7 +29,7 @@ REGISTER_TEST(negative_enqueue_map_image)
 {
     constexpr size_t image_dim = 32;
 
-    REQUIRE_EXTENSION("cl_ext_immutable_memory_objects");
+    REQUIRE_EXTENSION(CL_EXT_IMMUTABLE_MEMORY_OBJECTS);
```

Comment:
This is clever, but I wonder if it's a bit too clever, since things like intellisense linking and even grep'ing for these symbols won't work.  Should we just type out the extension name and extension version instead?

```c++
REQUIRE_EXTENSION(
    CL_EXT_IMMUTABLE_MEMORY_OBJECTS_EXTENSION_NAME,
    CL_EXT_IMMUTABLE_MEMORY_OBJECTS_VERSION);
```